integration_tests/test_rough_align.py

```diff
@@ -772,72 +956,6 @@ def test_setting_new_z_montage_scape(render, montage_stack, downsample_sections_
     assert(set(zvalues) == set([1020, 1021, 1022]))
 
 
-
-def test_solver_default_options(render, montage_scape_stack, rough_point_match_collection, tmpdir_factory):
```

Comment:
There are reasons to keep some compatibility with the MATLAB solver, even if it is not used in production (and even more reasons not to rewrite tests/dig through PRs).  For things like this, could we mark the test with a skip rather than delete it for the immediate future and follow up with a conversation about how we want to deprecate things?

Reply:
added the test back